### PR TITLE
PIM-331 Video Changes

### PR DESCRIPTION
**Pull request by mnzit**

### CODE REVIEW

File: src/main/java/life/genny/constants/QualityConstants.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class QualityConstants {
+
+    public static Map<String, Integer> quality = new HashMap<>();
+
+    static {
+        quality.put("360", 800000);
```
Comment (varunshastry) [inline src/main/java/life/genny/constants/QualityConstants.java]: The constants to be loaded from a "properties" file.

File: src/main/java/life/genny/utils/VideoQualityConverter.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            }
+        }
+        return true;
+    }
+
+    public static Boolean convert(File input, String fileName, Integer bitRate) {
+        try {
+            fileName = fileName.concat(".mp4");
+            File target = VideoUtils.convert(fileName, input, "mp4", bitRate);
+            log.debug("#### Converted: " + fileName);
```
Comment (varunshastry) [inline src/main/java/life/genny/utils/VideoQualityConverter.java]: #### to be removed

File: src/main/java/life/genny/utils/VideoUtils.java [hunk 3]
```diff
@@ -15,16 +16,14 @@
...
-    protected static final Logger log = org.apache.logging.log4j.LogManager.getLogger(VideoUtils.class);
-    public static File convert(File input, String videoType) throws IOException, EncoderException {
-        log.debug("#### Starting video conversion");
-        Instant start = Instant.now();
-        String fileName = UUID.randomUUID().toString();
-        File target = TemporaryFileStore.createTemporaryFile(fileName + ".mp4");
+    private static Logger log = LoggerFactory.getLogger(VideoUtils.class);
 
+    public static File convert(String fileName, File input, String videoType, Integer videoBitrate) throws IOException, EncoderException {
+        log.debug("#### Starting video conversion for: "+ fileName);
```
Comment (varunshastry) [inline src/main/java/life/genny/utils/VideoUtils.java]: #### to be removed

File: src/main/java/life/genny/utils/VideoUtils.java [hunk 4]
```diff
@@ -33,9 +32,9 @@ public static File convert(File input, String videoType) throws IOException, Enc
         VideoAttributes video = new VideoAttributes();
         video.setCodec("h264");
```
Comment (varunshastry) [inline src/main/java/life/genny/utils/VideoUtils.java]: Extract to constants

**Review by varunshastry — APPROVED** [review]

File: src/main/java/life/genny/utils/VideoQualityConverter.java (hunk 2 above)
Comment (mnzit) [reply src/main/java/life/genny/utils/VideoQualityConverter.java]: mate removing the # or entire log line

**Review by varunshastry — APPROVED** [review]

**Pull request merged**